ledger-tool/src/bigtable.rs

```diff
@@ -92,12 +94,71 @@ async fn blocks(starting_slot: Slot, limit: usize) -> Result<(), Box<dyn std::er
     Ok(())
 }
 
+async fn compare_blocks(
+    starting_slot: Slot,
+    limit: usize,
+    cred_path: Option<String>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    if let Some(credential) = &cred_path {
+        assert!(!credential.is_empty());
+    }
+
+    let bigtable_def = solana_storage_bigtable::LedgerStorage::new(false, None, None)
+        .await
+        .map_err(|err| format!("failed to connect to default bigtable: {:?}", err))?;
+
+    let slots_def = bigtable_def
+        .get_confirmed_blocks(starting_slot, limit)
+        .await?;
+    log::info!("default bigtable {} blocks found", slots_def.len());
+
+    let bigtable_std = solana_storage_bigtable::LedgerStorage::new(false, None, cred_path)
+        .await
+        .map_err(|err| format!("failed to connect to standard bigtable: {:?}", err))?;
+
+    let slots_std = bigtable_std
+        .get_confirmed_blocks(starting_slot, limit)
+        .await?;
+    log::info!("standard bigtable {} blocks found", slots_std.len());
+    let last = if slots_std.is_empty() {
+        serde_json::Number::from(-1)
```

Comment:
Hmm, not sure I love using `-1` for this. How about an Option instead, so that the json value is null if no blocks found?

Reply:
good idea